sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/FileSystemCertificates.java

```diff
@@ -170,45 +174,17 @@ public static String toCertificateAlias(File file) {
     }
 
     /**
-     * Factory of FileSystemCertificate, to avoid loading files multiple times
+     * Get File System certificates by path
+     *
+     * @param path certificate path, which works only in first time
+     * @return file certificate
      */
-    public static class FileSystemCertificatesFactory {
-
-        private static volatile FileSystemCertificates customFileSystemCertificates;
-
-        /**
-         * Get Singleton custom file system certificates
-         * @param path custom certificate path, which works only in first time
-         * @return custom file certificate
-         */
-        public static FileSystemCertificates getCustomFileSystemCertificates(String path) {
-            if (customFileSystemCertificates == null) {
-                synchronized (FileSystemCertificatesFactory.class) {
-                    if (customFileSystemCertificates == null) {
-                        customFileSystemCertificates = new FileSystemCertificates(path);
-                    }
-                }
-            }
-            return customFileSystemCertificates;
-        }
-
-        private static volatile FileSystemCertificates wellKnownFileSystemCertificates;
-
-        /**
-         * Get Singleton well known file system certificates
-         * @param path well known certificate path, which works only in first time
-         * @return well known file certificate
-         */
-        public static FileSystemCertificates getWellKnownFileSystemCertificates(String path) {
-            if (wellKnownFileSystemCertificates == null) {
-                synchronized (FileSystemCertificatesFactory.class) {
-                    if (wellKnownFileSystemCertificates == null) {
-                        wellKnownFileSystemCertificates = new FileSystemCertificates(path);
-                    }
-                }
-            }
-            return wellKnownFileSystemCertificates;
+    public static synchronized FileSystemCertificates getFileSystemCertificates(String path) {
```

Comment:
Because we have `synchronized` here, so we can change `CACHE`'s type from `ConcurrentHashMap` to `HashMap`.